lib/adapter/freshheads/CopyFilesToBuildDirAdapter.ts

```diff
@@ -66,15 +70,12 @@ export default class CopyFilesToBuildDirAdapter implements Adapter {
                 webpackConfig.plugins = [];
             }
 
-            const options = {
-                copyUnmodified: true,
```

Comment:
Dit moeten we wel even goed testen, dit loste ooit problemen op in combinatie met webpack asset helper. Maar nu is deze optie verwijderd dus geen idee of dit bij ons problemen gaat geven.

Reply:
Wel raar dat hier dan geen comment bij staat. Volgens mij is de issue dat de clean webpack plugin alle assets weggooit, en dan de oude assets, die niet modified zijn, niet meer terugkomen? Zou dat het kunnen zijn?

Reply:
Er is hier wel wat discussie over: https://github.com/webpack-contrib/copy-webpack-plugin/issues/497

> the copyUnmodified option was removed. This works well with Webpack 5, however with Webpack 4 all assets are always emitted during a watch build.

Correct me if i'm wrong, but i guess this was done to fix issues with the clean-webpack-plugin which removed stale assets by default, right? I recently worked on another plugin which also needed to copy a bunch of files and i used the same approach as this plugin by using the additionalAssets hook. What i did to there to prevent emitting unchanged assets was to use the internal existsAt. Webpack will automatically set this and all that is needed to make it work is storing the complete asset object instead of the raw content so that the value of existsAt will be persisted between builds in watch mode. For assets that didn't change the assets object can be added the asset list again as is. That way Webpack doesn't emit unchanged assets again and clean-webpack-plugin should also be happy since the asset is in the list.

> This is the expected behavior. By returning the option, we will again break other plugins. What exactly is the problem? 

Lijkt me toch even kwestie van testen dan. In mijn test project had ik geen afbeeldingen die hier eenvoudig mee te testen zijn.